recipes/MADL-CPU-OpenMPI/Data-Shredding/README.md

```diff
@@ -0,0 +1,27 @@
+## MADL-CPU-OpenMPI Data Shredding
+We included a python script that shows how to shred and deploy your training data prior to running a training job on Azure VMs via Open MPI.
```

Comment:
Avoid using "We". Perhaps reword as:

`This Data Shredding recipe shows how to shred`...